<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FBlurDialog</class>
 <widget class="QDialog" name="FBlurDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blur</string>
  </property>
  <widget class="QComboBox" name="cboType">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Uniform</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gaussian</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Median</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lblType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnKernelX">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblKernelX">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Kernel X</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblKernelY">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Kernel Y</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnKernelY">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblSigmaX">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sigma X</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spnSigmaX">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblSigmaY">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sigma Y</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spnSigmaY">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdOk">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdCancel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
